src/schemathesis/cli/__init__.py

```diff
@@ -37,6 +37,14 @@ def main() -> None:
     type=str,
     callback=validators.validate_auth,  # type: ignore
 )
+@click.option(  # type: ignore
```

Comment:
I am tending to `httpie` approach with `--auth-type` with two options `basic` (default) and `digest`. In this case, there will be no ambiguity with having two different auth passed as cmd options - there will be only one auth with type switching. What do you think?